tensorflow_addons/text/tests/crf_test.py

```diff
@@ -316,7 +316,7 @@ def test_crf_decode():
         expected_max_sequence = all_sequences[expected_max_sequence_index]
         expected_max_score = all_sequence_scores[expected_max_sequence_index]
 
-        actual_max_sequence, actual_max_score = text.crf_decode(
+        actual_max_sequence, _, actual_max_score = text.crf_decode(
```

Comment:
let's also test if the second return value, say `scores`, is correct.

Reply:
How would you approach this? In the test we compare the results of `crf_sequence_score` with `crf_decode`. Do I need to update `crf_sequence_score`?

Reply:
Correct!